bundles/org.openhab.binding.freeathomesystem/src/main/resources/OH-INF/i18n/freeathomesystem_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
This file shoud not be uploaded. Please use the localisation commands to generate these files.

Reply:
General localisation is added and placeholder is removed